### ISSUE TITLE
Shell: Does it really need to crash on each mistyped command?

### ISSUE THREAD

**Issue by vorou** [mbdavid/LiteDB#1013]
Found it pretty unusable, you have to re-run it and type "open" every time you made a mistake.
Is it by design?

**Comment by mbdavid** [comment]
Hi @vorou, is not work like that. You must run open only once you want open database. If you made any syntax error you got only exception about this

**Comment by vorou** [comment]
Well, what I see now is any exception thrown is crashing the process.
Could it be something on my side?

**Comment by rhullah** [comment]
I'm experiencing the same thing as @vorou, if I type a command in incorrectly the program exits with an exception.  Looking at the code, it seems like [the try/catch was commented out](https://github.com/mbdavid/LiteDB/blob/d78587d5d90d0e4b5b31caa24ac593cadb36c19f/LiteDB.Shell/Shell/ShellProgram.cs#L31..L57) that would have allowed it to continue running.

https://github.com/mbdavid/LiteDB/blob/d78587d5d90d0e4b5b31caa24ac593cadb36c19f/LiteDB.Shell/Shell/ShellProgram.cs#L31

**Comment by mbdavid** [comment]
Hi @rhullah, yes, this lines was comment during an debug and I forgot to revert. Thanks... I'm doing this now.

**Comment by lbnascimento** [comment]
Hi! With the objective of organizing our issues, we are closing old unsolved issues. Please check the latest version of LiteDB and open a new issue if your problem/question/suggestion still applies. Thanks!

**Comment by omfgicbf** [comment]
@lbnascimento how about you leave it open seeing as it's going to be fixed in v4.1.5, which is being released "soon".
